urdfpy/utils.py

```diff
@@ -199,6 +218,7 @@ def get_filename(base_path, file_path, makedirs=False):
         The resolved filepath -- just the normal ``file_path`` if it was an
         absolute path, otherwise that path joined to ``base_path``.
     """
+    file_path = resolve_filepath(base_path, file_path)
     fn = file_path
     if not os.path.isabs(file_path):
         fn = os.path.join(base_path, file_path)
```

Comment:
when given a relative path, this gives the wrong path. `resolve_filepath` returns `base_path + file_path`, then this if makes `fn = base_path + base_path + file_path`. Should this be `os.path.join(os.getcwd(), file_path)` instead?